### ISSUE TITLE
Minimizing the app make it go to the tray instead of staying on the Taskbar

### ISSUE THREAD

**Issue by JustCarousell** [getferdi/ferdi#882]
**Bug description:**

As the title mentioned. There's 2 ways that we can keep the app away from us which is, 
1. Clicking the X which is the close button will minimize the app to the tray (if the settings for this function is on)
2. Clicking the minimize button will minimize the app to the taskbar

so the issue here is on number two which instead of doing so to keep the app in the taskbar it acts as the same function as number one which is this function does not have to duplicate itself to another function.

**Steps to reproduce:**

**Expected behavior**

So when clicking minimize it should keep the app in the taskbar instead of the tray, as simple as that. :)

**Screenshots**

**Environment:**

- Operating System:  Windows 10
- Ferdi Version:  5.5.0
- Server:  Using without an account

- Debug information:

**Comment by vantezzen** [comment]
Have you tried disabling "Minimize Ferdi to system tray" in Ferdi's settings?

**Comment by JustCarousell** [comment]
Yup I tried that and then the close button now instead of keeping the app in the tray it actually work as minimize to the taskbar

**Comment by JustCarousell** [comment]
So basically how things should work is this.
1. When I click on close the app should go into the tray instead of minimizing to the taskbar.
2. When I click on minimize the app should minimize to the taskbar instead of going into the tray.

Problem facing now is both is working the same way instead of separately either they both work as minimize to taskbar or minimize to the tray which should not be the way.

**Comment by thedanioner** [comment]
> So basically how things should work is this.
> 
> 1. When I click on close the app should go into the tray instead of minimizing to the taskbar.
> 2. When I click on minimize the app should minimize to the taskbar instead of going into the tray.
> 
> Problem facing now is both is working the same way instead of separately either they both work as minimize to taskbar or minimize to the tray which should not be the way.
same problem

**Comment by dani-yisus** [comment]
I have the same problem too.

**Comment by Tsakatac** [comment]
Hello,

I think this issue should not be staled because I have the exact same issue also.
It only appears **after** closing it. At launch, I can minimize it in the taskbar.
My settings are like this.
![image](https://user-images.githubusercontent.com/89021195/129599150-506fa6ff-0e28-4ae5-ab25-9b5991ebd18d.png)

**Environment**:

- Operating System: Windows 10
- Ferdi Version: 5.6.0
![ferdi-version](https://user-images.githubusercontent.com/89021195/129599505-92ca7705-09a7-4ca7-a06c-90c2347c77ae.jpg)

Thanks!
